tests/mock/test_create_df_from_pandas.py

```diff
@@ -0,0 +1,222 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
```

Comment:
Shall we enable some tests in `tests/integ/test_pandas_to_df.py`?

Reply:
updated to enable test